codegen/render/manifests_renderer.go

```diff
@@ -37,9 +46,19 @@ func (r ManifestsRenderer) RenderManifests(grp Group) ([]OutFile, error) {
 	if !grp.RenderManifests {
 		return nil, nil
 	}
+
+	var err error
```

Comment:
just a nit, but in cases like this i like to ut the `var err` declaration inside the block where it's read